fastpair/rust/rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 80
```

Comment:
why so short?

Reply:
g3 rustfmt will reformat these if saved in cider or something so that's probably why.